### DISCUSSION TITLE
What is the time interval between each wallpaper change?

### DISCUSSION THREAD

**Discussion by sahibdsv** [t1m0thyj/WinDynamicDesktop#348] [Q&A] [answered]
I'm trying to replicate this project on Tasker so my Android's dynamic wallpaper "syncs" with my desktop.

I've already exported the high-res wallpapers onto the phone, and I've set it up to increase the 'wallpaper file name' by '1' every 90 mins.

I went through the engine but I can't find a _definite_ interval that I can use...

I saw that it splits each 24-hour solar cycle into 4 segments: sunrise, day, sunset, and night, but does every segment also have a kind of 'mini-segment' to allow showing all 16 variations of the wallpaper?

**Comment by t1m0thyj** [comment, marked as answer]
> I went through the engine but I can't find a definite interval that I can use...
> 
> I saw that it splits each 24-hour solar cycle into 4 segments: sunrise, day, sunset, and night

You're right that there is no definite interval, and it's also not hardcoded to 16. The number of images varies for each theme based on  how many are defined in the theme.json file. For more details about how the image scheduling works, see the [WDD wiki](https://github.com/t1m0thyj/WinDynamicDesktop/wiki/Creating-custom-themes#theme-file).
